src/client/scripts/esm/game/input.js

```diff
@@ -65,6 +67,10 @@ let ignoreMouseDown = false;
 
 let mouseIsSupported = true;
 
+/** Touchscreen and mouse */
+let usingTouchscreen; // True if the mouse recent pointer input was from a touch event.
```

Comment:
This variable name is a little confusing for what it means. Maybe something like `pointerIsTouch`? Is your intention that this will work on devices that have both a mouse AND touch-screen compatibility?

Reply:
The variable is for devices that support both. I wanted the piece to be offset upward only if the user is currently using the touchscreen. Is there a better way to do this?

Reply:
That's reasonable then to have a variable then to store whether the current pointer is a finger or not! I'll repeat the variable name was just a little confusing.